miden-crypto/benches/hash.rs

```diff
@@ -124,7 +124,7 @@ benchmark_hash_merge_many!(
             let data = generate_byte_array_sequential(64);
             digests.push(Rpo256::hash(&data));
         }
-        b.iter(|| Rpo256::merge_many(black_box(&digests)))
+        b.iter(|| <Rpo256 as AlgebraicSponge>::merge_many(black_box(&digests)))
```

Comment:
Should we remove the ambiguity by removing merge_many impls?

Reply:
Yes.